react/features/base/react/components/AbstractContainer.js

```diff
@@ -31,6 +31,12 @@ export type Props = {
      */
     onClick?: ?Function,
 
+    /**
+     * The event handler/listener to be invoked when this
+     * {@code AbstractContainer} is long pressed on React Native.
+     */
+    onLongPress?: ?Function,
```

Comment:
This needs to be moved to the native implementation. There is no long press on web and both share this base class.